src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyForIteratorPadCheck.java

```diff
@@ -27,26 +27,38 @@
 import com.puppycrawl.tools.checkstyle.api.TokenTypes;
 
 /**
- * <p>Checks the padding of an empty for iterator; that is whether a
- * space is required at an empty for iterator, or such spaces are
- * forbidden. No check occurs if there is a line wrap at the iterator, as in
+ * <p>
+ * Checks the padding of an empty for iterator; that is whether white
+ * space is required at an empty for iterator, or such white space is
+ * forbidden.  No check occurs if there is a line wrap at the iterator, as in
```

Comment:
Old:
> that is whether a space is required at an empty for iterator, or such spaces are forbidden.

New:
> that is whether white space is required at an empty for iterator, or such white space is forbidden.

New has more specifics like `white space` but old has better grammar like `a space`.
How about the following?
`that is whether a white space is required at an empty for iterator, or such white spaces are forbidden.`

Reply:
done.